test/gui/shared/scripts/helpers/api/Provisioning.py

```diff
@@ -58,3 +59,35 @@ def add_user_to_group(user, group_name):
         ocis.add_user_to_group(user, group_name)
     else:
         oc.add_user_to_group(user, group_name)
+
+
+def create_user(username):
+    user = {}
+    if username in userHelper.test_users:
+        user = userHelper.test_users[username]
+    else:
+        user = {
+            "username": username,
+            "displayname": username,
+            "email": f'{username}@mail.com',
+            "password": userHelper.test_users["regularuser"]["password"],
+        }
+
+    if get_config('ocis'):
+        user_info = ocis.create_user(
+            user['displayname'], user['email'], user['username'], user['password']
+        )
+    else:
+        user_info = oc.create_user(
+            user['username'], user['password'], user['displayname'], user['email']
+        )
+    userHelper.createdUsers[username] = user_info
+
+
+def delete_created_users():
+    for username, user_info in list(userHelper.createdUsers.items()):
+        if get_config('ocis'):
+            ocis.delete_user(user_info["id"])
+        else:
+            oc.delete_user(user_info["userid"])
```

Comment:
```suggestion
            oc.delete_user(user_info["username"])
```

Reply:
in oc10, userinfo contains `userid` but not `username`